deploy/charts/crate-operator-crds/README.md

```diff
@@ -0,0 +1,26 @@
+# Crate Operator CRDs Helm Chart
+
+A Helm chart for installing and upgrading the CRDs for [Crate Operator](https://github.com/crate/crate-operator).
+To be able to deploy the custom resource CrateDB to a Kubernetes cluster, the API needs to be extended with a Custom Resource Definition (CRD).
+
+Helm must be installed to use the charts. Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+## Usage
+
+Once Helm is properly set up, add the repo:
+
+```console
+helm repo add crate https://
```

Comment:
Should we document installing from the local folder?

Reply:
I waited to have set up the github action to publish the chart before competing the repo url here, it should be `https://crate.github.io/crate-operator` in theory.
I would rather document using the repo instead of the local folder as the local method is supposed to me more for testing and stuff like that.